tests/functional/create_subnets_test.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2018 Cable Television Laboratories, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from drb_python.http_exceptions import AuthorizationError, ConnectionError
+from drb_python.drb_exceptions import ActionError, AlreadyExists
+from drb_python.subnet import Subnet
+from drb_python.http_session import HttpSession
+
+
+class SubnetTest(unittest.TestCase):
+    """
+    Tests for functions located in SubnetHttps
+    """
+    def test_create_subnet(self):
+        login = {'username': 'rocketskates', 'password': 'r0cketsk8ts'}
+        subnet_object = {
+            'address': '10.197.111.0',
```

Comment:
We'll need some sort of construct that on can use to pass in this variable data. Please see the component and integration tests within snaps-oo for one example on how to do this.